docs/plugin_fundamentals/import_automation.md

```diff
@@ -62,11 +61,11 @@ namespace Zeiss.SecondImportAutomation;
 
 public class Plugin : IPlugin
 {
-  public IImportAutomation CreateImportAutomation(ICreateImportAutomationContext context)
-  {
-    // Registration of a new instance of IImportAutomation with Auto Importer
-    return new ImportAutomation();
-  }
+    public IImportAutomation CreateImportAutomation(ICreateImportAutomationContext context)
```

Comment:
We have decided to only indent with 2 spaces to keep the code more compact and clearer. Would be great if you could revert that.

Reply:
Oops...

Reply:
Well that was fun...